glimmer-v0.7.2-keyed/config/targets.js

```diff
@@ -0,0 +1,12 @@
+let browsers = [
+  'last 1 Chrome versions'
```

Comment:
Re: https://github.com/krausest/js-framework-benchmark/issues/243 I pinned this to only support the most recent version of Chrome.